delivery.yaml

```diff
@@ -18,9 +20,11 @@ pipeline:
     cmd: |
       if [[ $CDP_TARGET_BRANCH == master && ! $CDP_PULL_REQUEST_NUMBER ]]; then
         IMAGE=registry-write.opensource.zalan.do/teapot/kube-metrics-adapter
-        VERSION=$(git describe --tags --always --dirty)
+        VERSION=$(git describe --tags --exact)
```

Comment:
Does this mean we can't build master images for non-tagged versions? It's IMO convenient to be able to try an image out before tagging the version.

Reply:
Shouldn't the PR versions be enough for this?

Reply:
I don't mind reverting, though, but I'd still like to keep the `--dirty` part dropped.

Reply:
Fixed.